packages/hoprd/src/commands/closeChannel.ts

```diff
@@ -17,33 +17,35 @@ export default class CloseChannel extends AbstractCommand {
     return 'Close an open channel'
   }
 
-  async execute(query: string, state: GlobalState): Promise<string | void> {
+  async execute(log, query: string, state: GlobalState): Promise<void> {
     if (query == null) {
-      return styleValue(`Invalid arguments. Expected 'close <peerId>'. Received '${query}'`, 'failure')
+      return log(styleValue(`Invalid arguments. Expected 'close <peerId>'. Received '${query}'`, 'failure'))
     }
 
     let peerId: PeerId
     try {
       peerId = await checkPeerIdInput(query, state)
     } catch (err) {
-      return styleValue(err.message, 'failure')
+      return log(styleValue(err.message, 'failure'))
     }
 
     try {
       const { status, receipt } = await this.node.closeChannel(peerId)
 
-      if (status === 'PENDING') {
```

Comment:
we need to check: if pending, else if open, else nothing

Reply:
at present if it isn't OPEN, initiateChannelClosure throws, thus we get the error message here.